test/extended/operators/olm.go

```diff
@@ -105,4 +105,18 @@ var _ = g.Describe("[Feature:Platform] OLM should", func() {
 		}
 	})
 
+	// OCP-24829 - Report `Upgradeable` in OLM ClusterOperators status
+	// author: bandrade@redhat.com
+	g.It("Report Upgradeable in OLM ClusterOperators status", func() {
+		olmCOs := []string{"operator-lifecycle-manager", "operator-lifecycle-manager-catalog", "operator-lifecycle-manager-packageserver"}
+		for _, co := range olmCOs {
+			msg, err := oc.AsAdmin().WithoutNamespace().Run("get").Args("co", co, "-o=jsonpath={range .status.conditions[*]}{.type}").Output()
```

Comment:
Is the objective of this test to check whether OLM writes _any_ upgradeable status, or that it _is_ upgradeable? In the later case, you would also need to ensure the condition's `status` field is set `True`.

Reply:
Fixed as requested